src/TagLib/Ogg/XiphComment.cs

```diff
@@ -1409,7 +1409,15 @@ public override string[] PerformersRole
 			}
 			set
 			{
-				pictures = value;
+				if (value == null) {
+					// Set pictures to a 0-length array to prevent
+					// re-parsing by the getter on the next access
+					pictures = new IPicture[0];
```

Comment:
It is probably better to use `Enumberable.Empty<IPicture>()` instead of `new IPicture[0]`

Reply:
Enumerable.Empty<T>() returns an IEnumerable<T>, so it can't be used for the pictures field.

Reply:
oh ok, There is also an `Array.Empty<IPicture>()`. I was just trying to be more abstract ;)

Reply:
I didn't know about `Array.Empty<T>()`, but :+1: for abstraction ;-)

Reply:
Actually `Array.Empty<T>()` is .NET Framework 4.6 (source: https://msdn.microsoft.com/fr-fr/library/dn906179(v=vs.110).aspx) so we'd have to drop .NET Framework 4 for that method...

Reply:
oh, too bad. It might be worth caching off the `new IPicture[0]` for now then. That is what `Array.Empty<T>()` is doing

Reply:
Done in 701a2c14089a0d9e0c51af63fc0c5176c1291e1c